packages/components/addon/components/hds/code-block/index.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { assert } from '@ember/debug';
+import { htmlSafe } from '@ember/template';
+import Prism from 'prismjs';
+import { setup } from 'prismjs-glimmer';
+
+import 'prismjs/plugins/line-numbers/prism-line-numbers';
+import 'prismjs/plugins/line-highlight/prism-line-highlight';
+
+// Importing language individually because autoloader isn't currently working
+import 'prismjs/components/prism-go';
+import 'prismjs/components/prism-markdown';
+import 'prismjs/components/prism-shell-session';
+
+// languages_path isn't working so autoloader isn't working
+// https://prismjs.com/plugins/autoloader/
+import 'prismjs/plugins/autoloader/prism-autoloader';
```

Comment:
Getting 404s when it tries to link to the language files
<img width="1074" alt="Screenshot 2023-08-31 at 9 20 07 AM" src="https://github.com/hashicorp/design-system/assets/108769823/f6a88ba4-a64f-4847-b1ff-da9064518fd7">

Reply:
probably they should live in the `public/` folder?

Reply:
https://prismjs.com/plugins/autoloader/ → "Specifying the grammars path" maybe?

Reply:
I already tried specifying the path explicitly. I can experiment a bit more though.